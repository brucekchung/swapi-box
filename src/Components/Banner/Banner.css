.Banner-header {
  background-color: black;
  height: 100px;
  padding: 10px;
  color: white;
  z-index: 2
}

.Banner-header h1 {
  font-size: 1.5em;
  text-align: center;
}

.Banner-header img {
  height: 90px;
  width: 180px;
  margin: 0 auto;
  display: block;
}