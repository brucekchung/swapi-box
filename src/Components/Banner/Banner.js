import React from 'react'
import './Banner.css'
import logo from '../../assets/images/starwars-logo.png'

const Banner = () => (
  <div className="Banner-header">
    <img src={logo} alt="starwars-logo" />
    <h1>API box</h1>
  </div>
)

export default Banner