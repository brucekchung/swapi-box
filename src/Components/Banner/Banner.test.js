import React from 'react'
import { shallow } from 'enzyme'
import Banner from './Banner'

describe('Banner', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<Banner />)
    
    expect(wrapper).toMatchSnapshot()
  })
})