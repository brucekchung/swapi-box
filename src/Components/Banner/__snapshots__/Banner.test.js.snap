// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Banner />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        ariaCurrent="true"
        to="/"
>
        <img
                alt="starwars-logo"
                src="starwars-logo.png"
        />
</NavLink>,
      "className": "Banner-header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "ariaCurrent": "true",
        "children": <img
          alt="starwars-logo"
          src="starwars-logo.png"
/>,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "starwars-logo",
          "src": "starwars-logo.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/"
>
          <img
                    alt="starwars-logo"
                    src="starwars-logo.png"
          />
</NavLink>,
        "className": "Banner-header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <img
            alt="starwars-logo"
            src="starwars-logo.png"
/>,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "starwars-logo",
            "src": "starwars-logo.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
